zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -315,7 +315,7 @@ class Handlers(mustacheGenerator: ZipkinMustache, queryExtractor: QueryExtractor
         "parentId" -> span.parentId.filter(spanMap.get(_).isDefined).map(SpanId(_).toString),
         "spanName" -> span.name,
         "serviceNames" -> span.serviceNames.mkString(","),
-        "serviceName" -> span.serviceName,
+        "serviceName" -> span.serviceName.getOrElse(""),
```

Comment:
guess what I'm wondering is ...
 - if we known we won't return spans who have no service name..
 - and we fix prioritization to deprioritize empty string..
wouldn't it be a bug for this to be null (or empty)?